src/Mado/QueryBundle/Queries/Options/QueryOptionsBuilder.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+namespace Mado\QueryBundle\Queries\Options;
+
+use Mado\QueryBundle\Exceptions\InvalidFiltersException;
+use Mado\QueryBundle\Queries\QueryBuilderOptions;
+use Symfony\Component\HttpFoundation\Request;
+
+class QueryOptionsBuilder
+{
+    private $entityAlias;
+
+    public function setEntityAlias(string $entityAlias)
+    {
+        $this->entityAlias = $entityAlias;
+    }
+
+    public function getEntityAlias()
+    {
+        return $this->entityAlias;
+    }
+
+    public function builderFromRequest(Request $request = null)
```

Comment:
I suggest to change the name to buildFromRequest as the others methods, what do you think?

Reply:
👍